policyengine_us/parameters/gov/ssa/ssi/income/exclusions/blind_or_disabled_student/cap.yaml

```diff
@@ -0,0 +1,18 @@
+description: Yearly cap of earned income excluded from SSI countable income for blind or disabled working student.
+
+values:
+  2011-01-01: 6_600
```

Comment:
do we have the exact values from 2011 to now? If it is uprated with inflation then there should be values in tax forms

Reply:
> do we have the exact values from 2011 to now? If it is uprated with inflation then there should be values in tax forms

the annual ssi reports have the updated values, do we want to include every one of these (for every year) or just the most recent?

Reply:
> do we have the exact values from 2011 to now? If it is uprated with inflation then there should be values in tax forms

the annual SSI reports include yearly values, should I just put the most recent value for both parameters?

Reply:
Lets keep the 2011 and add as many values as we can find from then to now